### PR TITLE
clib@next

### PR DESCRIPTION
**Pull request by jwerle**
Getting this started, but still a WIP!

### CODE REVIEW

File: test/install-binary-dependencies.sh [hunk 1]
```diff
@@ -4,7 +4,7 @@ clib install -c stephenmathieson/tabs-to-spaces@1.0.0 > /dev/null || {
   echo >&2 "Failed to install stephenmathieson/tabs-to-spaces"
   exit 1
 }
-command -v t2s >/dev/null 2>&1 || {
+command -v $PREFIX/bin/t2s >/dev/null 2>&1 || {
```
Comment (vipyne) [inline test/install-binary-dependencies.sh:7]: This breaks the test for me on OSX (and also linux?).  What was it supposed to fix?

File: src/clib-init.c [hunk 2]
```diff
@@ -1,7 +1,7 @@
 //
 // clib-init.c
 //
-// Copyright (c) 2012-2014 clib authors
+// Copyright (c) 2012-2019 clib authors
```
Comment (vipyne) [inline src/clib-init.c:4]: 2020 ?

File: test/install-binary-dependencies.sh (hunk 1 above)
Comment (jwerle) [reply test/install-binary-dependencies.sh:7]: looks like a test for installing a binary command and its deps, what is breaking on your end?

File: src/clib-init.c (hunk 2 above)
Comment (jwerle) [reply src/clib-init.c:4]: :] yep!

File: test/install-binary-dependencies.sh (hunk 1 above)
Comment (vipyne) [reply test/install-binary-dependencies.sh:7]: I get `Failed to put t2s on path` when using the `$PREFIX/bin/` version.
`command -v t2s >/dev/null 2>&1` version passes.  
Does a $PREFIX env var need to be set somewhere?

Comment (jwerle) [reply test/install-binary-dependencies.sh:7]: probably. I'll take a look!

**Pull request merged**